os/board/rtl8721csm/src/Makefile

```diff
@@ -133,12 +157,33 @@ CFLAGS += -I$(COMPONENT_DIR)/os/os_dep/include
 CFLAGS += -I$(COMPONENT_DIR)/os/tizenrt
 CFLAGS += -I$(TOPDIR)/include/net
 CFLAGS += -I$(TOPDIR)/net/lwip
-CFLAGS += -I$(TOPDIR)/net/lwip/src/include/
+CFLAGS += -I$(TOPDIR)/net/lwip/src/include
 CFLAGS += -I$(DIR_WIFI)/rtw_wpa_supplicant/src
 CFLAGS += -I$(DIR_WIFI)/rtw_wpa_supplicant/wpa_supplicant
 CFLAGS += -I$(COMPONENT_DIR)/soc/realtek/amebad/misc
-CFALGS += -I$(DIR_CRYPTO)/../include/
-CFLAGS += -I$(DIR_FW)/../include/
+CFALGS += -I$(DIR_CRYPTO)/../include
+CFLAGS += -I$(DIR_FW)/../include
+ifeq ($(CONFIG_AMEBAD_BLE),y)
+CFLAGS += -I$(DIR_BT)/board/amebad/lib
+CFLAGS += -I$(DIR_BT)/board/amebad/src
+CFLAGS += -I$(DIR_BT)/board/amebad/src/vendor_cmd
+CFLAGS += -I$(DIR_BT)/board/common/inc
+CFLAGS += -I$(DIR_BT)/example/ble_central
+CFLAGS += -I$(DIR_BT)/example/ble_peripheral
+CFLAGS += -I$(DIR_BT)/inc
+CFLAGS += -I$(DIR_BT)/inc/app
+CFLAGS += -I$(DIR_BT)/inc/bluetooth/gap
+CFLAGS += -I$(DIR_BT)/inc/bluetooth/profile
+CFLAGS += -I$(DIR_BT)/inc/bluetooth/profile/client
+CFLAGS += -I$(DIR_BT)/inc/bluetooth/profile/server
+CFLAGS += -I$(DIR_BT)/inc/os
+CFLAGS += -I$(DIR_BT)/inc/platform
+CFLAGS += -I$(DIR_BT)/inc/stack
+CFLAGS += -I$(DIR_BT)/src/mcu/module/data_uart_cmd
+CFLAGS += -I$(DIR_BT)/example/da_ble_peripheral
+CFLAGS += -I$(DIR_BT)/example/da_ble_central
+CFLAGS += -I$(TOPDIR)/../da_include
```

Comment:
You already have too much long CFLAG path. As you know, there is a limitation in CFLAG length, 4096. I worry about this limitation. Could you confirm that there is no cflag failure with CONFIG_AMEBAD_BLE enabled?

Reply:
The cflag failure do exist when CONFIG_AMEBAD_BLE enabled. But we found it seems to have no effect before. Do we need to fix cflag failure in your opnion？

Reply:
Sure. Please resolve the failure.